modules/appeals_api/spec/docs/v2/sc_spec.rb

```diff
@@ -159,6 +159,134 @@
     end
   end
 
+  path '/supplemental_claims/schema' do
+    get 'Gets the Supplemental Claims JSON Schema.' do
+      tags 'Supplemental Claims'
+      operationId 'scSchema'
+      description 'Returns the [JSON Schema](https://json-schema.org/) for the `POST /supplemental_claims` endpoint.'
+      security [
+        { apikey: [] }
+      ]
+      produces 'application/json'
+
+      response '200', 'the JSON Schema for POST /supplemental_claims' do
+        before do |example|
+          submit_request(example.metadata)
+        end
+
+        after do |example|
+          example.metadata[:response][:content] = {
+            'application/json' => {
+              example: JSON.parse(response.body, symbolize_names: true)
+            }
+          }
+        end
+
+        it 'returns a 200 response' do |example|
+          assert_response_matches_metadata(example.metadata)
+        end
```

Comment:
Sorry if I missed this last time, but I think this can also be reduced down with the `rspec example` behavior.

Reply:
on it